docs/azure/get-started-with-azure/guardrails.md

```diff
@@ -0,0 +1,220 @@
+
+Think of these guardrails as automatic checks and balances. They don't require you to do anything extra most of the time, but they might prevent you from doing something that goes against our established best practices, or they might automatically configure resources to meet compliance standards.
+
+!!! info
+    This document focuses on the things that you might be *prevented* from doing (Deny policies), the things you *must* do as a result, or things that will be automatically configured (DeployIfNotExists and Modify policies). Audit policies are also in place, which will flag non-compliant resources, but won't block deployments.
+
+## Key areas covered
+
+1.  **Resource Deployment Restrictions:** Where and what you can deploy.
+2.  **Networking Restrictions:** How your networks are configured and protected.
+3.  **Security Requirements:** Mandatory security settings.
+4.  **Data Protection:** How your data is protected at rest and in transit.
+5.  **Cost Optimization:** Preventing unnecessary costs.
+6.  **Identity & Access Management:** Controlling who can access resources.
+7.  **Specific Service Guardrails:** Restrictions related to specific Azure services.
+8.  **Monitoring:** Diagnostic data collection and automated security checks.
+9.  **Tagging:** Automatic application of tags.
+
+## Resource deployment restrictions
+
+*   **Allowed Locations (Regions):** You can only deploy resources in Canada Central and Canada East. If you try to deploy to a region not on the approved list, the deployment will be **blocked**. Note that networking is only provided in Canada Central.
+*   **Resource Group Locations:** The location (region) of a resource group *must* match the location of the resources within it.
+*   **Denied Resource Types:** There is a "Deny" policy for specific resource types that are *not* allowed within the landing zone, preventing their creation.
+
+## Networking restrictions
+
+### General network security
+*   **No Public IPs (Generally):** You *cannot* create public IP addresses directly. This is a crucial security measure to minimize our public attack surface. Services should typically be accessed through private endpoints or other secure means.
+*   **No Public IP on NICs:** Network interfaces *cannot* have a public IP address associated.
+*   **No Management Port Access from Internet:** NSG rules allowing inbound access from the "Internet" source on common management ports (SSH/RDP - ports 22 and 3389) are *blocked*. Access to these ports is only allowed through secure, managed pathways (e.g., Azure Bastion, Just-In-Time VM Access).
+
+### Subnet configuration requirements
```

Comment:
We should also include Private Subnets, as this was a new Azure Policy in one of the recent CAF updates.
- https://learn.microsoft.com/en-us/azure/virtual-network/ip-services/default-outbound-access#how-can-i-transition-to-an-explicit-method-of-public-connectivity-and-disable-default-outbound-access